website/docs/r/vpn_gateway.html.markdown

```diff
@@ -15,6 +15,8 @@ Provides a VPN gateway resource.
 
 -> Currently International-Site account can open `PostPaid` VPN gateway and China-Site account can open `PrePaid` VPN gateway.
 
+-> **NOTE:** If the resource is `PerPaid` type the resource Cannot destroy. Terraform will remove this resource from the state file, however resources may remain.
```

Comment:
-> **NOTE:** `PrePaid` VPN gateway cannot  be destroyed and provider will remove it from the state file.